styles/style.css

```diff
@@ -1,54 +1,95 @@
-.main {
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
   display: flex;
```

Comment:
- [ ] You are using `display: flex` 9 times, which is bad considering that you are repeating yourself a lot. I would suggest you create a class `flex` with `display: flex` and use it whenever you need